var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -805,9 +805,22 @@ def find_optional_library(name, prefix):
                 "apf_zoltan",
                 "spr",
             ]
+            pumi_dep_zoltan = [""]
+            pumi_dep_parmetis = [""]
```

Comment:
I think these should be initialized with `""` and not `[""]` because running the following in python:
```console
a=[""]
print("%s" % a)
```
prints
```console
['']
```
and we want it to produce the empty string.

Reply:
Fixed.  Good catch.  Thank you.